build.py

```diff
@@ -0,0 +1,86 @@
+import glob
+import os
+import shutil
+import urllib.request
+import zipfile
+
+import static_assets
+
+
+def build_govuk_assets(static_dist_root="app/static/dist"):
+    GOVUK_FRONTEND_VERSION = "4.6.0"
```

Comment:
The most up-to-date version is actually ```4.7.0```. This is the version in the Submit repo. Do we want to standardise the two?

Reply:
The latest version is actually 5.2.0! https://github.com/alphagov/govuk-frontend/releases

We could standardise it, just wasn't strictly required to upgrade for this change, so maybe could be picked up separately?

Reply:
@Joel-Pearce for now I kept this as per the previous build script: https://github.com/communitiesuk/funding-service-design-post-award-data-frontend/blob/8a22d458c9015fe973d0ccc938f75d05e4b24d03/build.sh#L7 maybe we could go up to the latest `4.x` release which shouldn't introduce breaking changes, but it's not strictly required for this change so didn't want to cause a change in behaviour inadvertently.